hack/lib/golang.sh

```diff
@@ -750,6 +750,29 @@ kube::golang::get_physmem() {
   echo 1
 }
 
+# Build unstripped binary on-demand
+# return value
+#    0: build unstripped binary
+#    1: build stripped binary
+kube::golang::build_unstripped_binary() {
+  local nflag lflag retval
+
+  nflag=0
+  lflag=0
+  retval=1
+
+  # Once "-N" and "-l" are specified together,
+  # binaries are built as unstripped
+  for flag in ${GOGCFLAGS:-}; do
+    [[ "${flag}" = "-N" ]] && nflag=1
+    [[ "${flag}" = "-l" ]] && lflag=1
```

Comment:
where do `-N` and `-l` come from. i can't seem to find them under:
https://golang.org/cmd/link/

Reply:
@neolit123 

    The user specified in GOGCFLAGS and please refer the explaination in Makefile from Line 81 to Line 84

https://github.com/kubernetes/kubernetes/blob/master/build/root/Makefile#L81~L84